protobufs/livekit_models.proto

```diff
@@ -227,7 +233,10 @@ message SimulcastCodecInfo {
 message TrackInfo {
   string sid = 1;
   TrackType type = 2;
-  string name = 3;
+  string name = 3 [
```

Comment:
does the size of these matter?

Reply:
actually no, just using the same construct everywhere. Will remove `redact_format` here.